internal/infrastructure/db/migrations/20220919141609_device_integration_features.sql

```diff
@@ -0,0 +1,13 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
```

Comment:
By the way, these `SELECT 'up SQL query';` lines don't do anything. I think a lot of people leave them in, though.